### PR TITLE
Add OCM link to Cluster Settings page

### PR DESCRIPTION
**Pull request by spadgett**
Screen Shot 2019-09-05 at 1 25 15 PM

/assign @rhamilto 

cc @smarterclayton @luciddreamz

### CODE REVIEW

**Comment by spadgett** [comment]
/hold
i think the link is wrong

File: frontend/public/components/cluster-settings/cluster-settings.tsx [hunk 1]
```diff
@@ -187,9 +189,17 @@ export const ClusterVersionDetailsTable: React.SFC<ClusterVersionDetailsTablePro
         </div>
       </div>
       <div className="co-m-pane__body-group">
+        <p className="co-m-pane__explanation">
+          View this cluster and manage subscription settings at
+          {' '}
+          <ExternalLink
+            text="cloud.redhat.com"
+            href={`https://cloud.redhat.com/openshift/details/${clusterID}`}
```
Comment (spadgett) [inline frontend/public/components/cluster-settings/cluster-settings.tsx]: @luciddreamz is there a way we can link directly to the cluster details? it looks like this link with the cluster ID doesn't work

Comment (gregsheremeta) [reply frontend/public/components/cluster-settings/cluster-settings.tsx]: it doesn't work as of today, but I will make it work :smile:

**Comment by spadgett** [comment]
Cluster link that uses the external URL will be added in SDA-1389

**Comment by ncameronbritt** [comment]
Could we make the link part "OpenShift Cluster Manager" rather than cloud.redhat.com? 

And maybe it already does, but the link should go directly to the details page for that cluster in OCM.

**Comment by spadgett** [comment]
Yeah, we're linking directly to the cluster details page. I can change the link text.

**Comment by spadgett** [comment]
Text updated in #2800

**Pull request merged**